### DISCUSSION TITLE
SHA256 failure on doxygen-1.9.3, every time

### DISCUSSION THREAD

**Discussion by mcclure** [Homebrew/discussions#2726] [Everyday usage] [answered]
I am running Homebrew on MacOS 10.13.6 on a new MacBook. When I run `brew upgrade` it fails with
```
==> Downloading https://doxygen.nl/files/doxygen-1.9.3.src.tar.gz
==> Downloading from https://www.doxygen.nl/files/doxygen-1.9.3.src.tar.gz
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 1a413e7122a0f9bb519697613ba05247afad1adc5699390b6e6ee387d42c0b2d
  Actual: f352dbc3221af7012b7b00935f2dfdc9fb67a97d43287d2f6c81c50449d254e0
    File: /Users/mcc/Library/Caches/Homebrew/downloads/cae0381a57f446d538ab4eb8ad46e4d747f053dc86dd4de28d4ac7c639cd0fb3--doxygen-1.9.3.src.tar.gz
To retry an incomplete download, remove the file above.
```
It does this every time. I have deleted the file twice and rerun brew upgrade and I get the same failure with the SHA mismatch. I ran brew update and tried again and got again the same failure.

I understand that I am on an unsupported configuration but it seems like this must be a failure on the server side.

I went to [file a bug](https://github.com/Homebrew/homebrew-core/issues/new/choose) and it said to only file if it is reproducible. I do not know how to check if this is reproducible on my own.

When I run `brew -v` it prints

```
Homebrew 3.3.9-129-g1105859
Homebrew/homebrew-core (git revision 0ad781ef4e5; last commit 2022-01-05)
Homebrew/homebrew-cask (git revision e38e9459d9; last commit 2022-01-05)
```

**Comment by gromgit** [comment]
From [their download page](https://www.doxygen.nl/download.html):

> The latest version of doxygen is 1.9.3 (release date 31 december 2021, updated 3th of January 2022).

I'm reaching out to the doxygen folks to determine the nature of the update, so as to decide whether we just need to change the formula checksum, or force a bottle rebuild.

**Comment by gromgit** [comment, marked as answer]
Fix in progress: https://github.com/Homebrew/homebrew-core/pull/92552

**Comment by mcclure** [comment]
As of this second I do not see the error anymore when running brew upgrade. Thanks!
